src/promptflow-evals/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features Added
 - Introduced `JailbreakAdversarialSimulator` for customers who need to do run jailbreak and non jailbreak adversarial simulations at the same time. More info in the README.md in `/promptflow/evals/synthetic/README.md#jailbreak-simulator`
+- Exposed batch evaluation run timeout via "PF_BATCH_TIMEOUT_SEC" environment variable

```

Comment:
Can we please specify this will be for each evaluator and target separately and not as a whole evaluate API timeout ?

Reply:
done